{{cookiecutter.project_slug}}/production.yml

```diff
@@ -3,67 +3,88 @@ version: '3'
 volumes:
   production_postgres_data: {}
   production_postgres_data_backups: {}
+  production_redis_data: {}
   production_caddy: {}
 
 services:
   django:{% if cookiecutter.use_celery == 'y' %} &django{% endif %}
     build:
       context: .
-      dockerfile: ./compose/production/django/Dockerfile
+      dockerfile: ./docker/django/Dockerfile
+      args:
+        environment: production
+        cmd: /django
     image: {{ cookiecutter.project_slug }}_production_django
     depends_on:
       - postgres
       - redis
     env_file:
-      - ./.envs/.production/.django
-      - ./.envs/.production/.postgres
-    command: /start
+      - ./.envs/.django
+      - ./.envs/.postgres
+    command: /django
+  {%- if cookiecutter.use_celery == 'y' %}
 
-  postgres:
+  celeryworker:
+    <<: *django
+    build:
+      context: .
+      dockerfile: ./docker/django/Dockerfile
+      args:
+        environment: production
+        cmd: /celeryworker
+    image: {{ cookiecutter.project_slug }}_production_celeryworker
+    ports: []
+    command: /celeryworker
+
+  celerybeat:
+    <<: *django
     build:
       context: .
-      dockerfile: ./compose/production/postgres/Dockerfile
+      dockerfile: ./docker/django/Dockerfile
+      args:
+        environment: production
+        cmd: /celerybeat
+    image: {{ cookiecutter.project_slug }}_production_celerybeat
+    user: root  # TODO: a temporary workaround -- do it right instead
```

Comment:
I'm sorry if that's a stupid question, but I didn't understand why it's required now and not before. I thought we were already running the celery beat with the Django image, which is using a non-root user in prod... Is it a pre-existing issue?